packages/toolboxes/substrate/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "author": "swapkit-oss",
   "dependencies": {
-    "@polkadot/api": "10.12.6",
-    "@polkadot/api-base": "10.12.6",
+    "@polkadot/api": "12.1.1",
+    "@polkadot/api-base": "12.1.1",
     "@polkadot/keyring": "12.6.2",
     "@polkadot/util": "12.6.2",
     "@polkadot/util-crypto": "12.6.2",
     "@swapkit/helpers": "workspace:*"
   },
   "description": "SwapKit - Toolbox substrate",
   "devDependencies": {
-    "@polkadot/types": "10.12.6"
+    "@polkadot/types": "12.1.1"
   },
```

Comment:
If it's used as `devDependencies` there - ensure that's included in `substrate/build.ts` file within `dependencies` as:
```js
import { dependencies, devDependencies } from './package.json'

build({ dependencies: { ...dependencies, ...devDependencies } })
```

As it shouldn't be included in build

Reply:
done